sdk/vision/azure-ai-vision-imageanalysis/azure/ai/vision/imageanalysis/_patch.py

```diff
@@ -35,41 +27,73 @@ class ImageAnalysisClient(ImageAnalysisClientGenerated):
     :paramtype api_version: str
     """
 
-    @overload
-    def analyze(
+    @distributed_trace
+    def analyze_from_url(
         self,
-        *,
         image_url: str,
         visual_features: List[_models.VisualFeatures],
-        language: Optional[str] = None,
-        gender_neutral_caption: Optional[bool] = None,
-        smart_crops_aspect_ratios: Optional[List[float]] = None,
-        model_version: Optional[str] = None,
-        **kwargs: Any
-    ) -> _models.ImageAnalysisResult:
-        ...
-
-    @overload
-    def analyze(
-        self,
         *,
-        image_data: bytes,
-        visual_features: List[_models.VisualFeatures],
         language: Optional[str] = None,
         gender_neutral_caption: Optional[bool] = None,
         smart_crops_aspect_ratios: Optional[List[float]] = None,
         model_version: Optional[str] = None,
         **kwargs: Any
     ) -> _models.ImageAnalysisResult:
-        ...
+        """Performs a single Image Analysis operation.
+
+        :keyword image_url: The publicly accessible URL of the image to analyze.
+        :paramtype image_url: str
+        :keyword visual_features: A list of visual features to analyze. Required. Seven visual features
+         are supported: Caption, DenseCaptions, Read (OCR), Tags, Objects, SmartCrops, and People. At
+         least one visual feature must be specified.
+        :paramtype visual_features: list[~azure.ai.vision.imageanalysis.models.VisualFeatures]
```

Comment:
image_url and visual_features should be documented as param/type now that they are required/positional:

```suggestion
        :param image_url: The publicly accessible URL of the image to analyze.
        :type image_url: str
        :param visual_features: A list of visual features to analyze. Required. Seven visual features
         are supported: Caption, DenseCaptions, Read (OCR), Tags, Objects, SmartCrops, and People. At
         least one visual feature must be specified.
        :type visual_features: list[~azure.ai.vision.imageanalysis.models.VisualFeatures]
```

Reply:
Oh, good catch! thank you, fixing it now.